src/main/java/legalthings/lto_api/lto/core/Account.java

```diff
@@ -0,0 +1,260 @@
+package legalthings.lto_api.lto.core;
+
+import legalthings.lto_api.utils.core.JsonObject;
+import legalthings.lto_api.utils.main.CryptoUtil;
+import legalthings.lto_api.utils.main.StringUtil;
+
+public class Account {
+	/**
+	 * Account public address
+	 * @var string
+	 */
+	public byte[] address;
+	
+	/**
+	 * Sign kyes
+	 * @var object
+	 */
+	public JsonObject sign;
+	
+	/**
+	 * Encryption keys
+	 * @var object
+	 */	
+	public JsonObject encrypt;
```

Comment:
Reuse the `KeyPair` class for there keys

Reply:
OK